### ISSUE TITLE
Infinite recursion when registering "blur" handler on text node in 3.4.0 & 3.4.1

### ISSUE THREAD

**Issue by xkr47** [jquery/jquery#4588]
### Description ###
3.4.0 and 3.4.1 contain a regression where registering a "blur" event handler on a text node causes infinite recursion. Aka "Uncaught RangeError: Maximum call stack size exceeded" or "InternalError: too much recursion".

Sample failing test code:
```js
$($('<div>foo</div>')[0].childNodes[0]).blur(() => 0)
```

### Link to test case ###

https://jsbin.com/tadocopure/edit?html,js,output

If one replaces `jquery-3.4.1` with `jquery-3.x-git` or `jquery-git` then it seems to work, so apparently fixed in development branches. Can we have a 3.x release?

**Comment by mgol** [comment]
Yes, this was fixed in #4558. We still have a few issues, also related to 3.4 focus changes that we'd like to fix in 3.5.0: https://github.com/jquery/jquery/milestone/21

I'll close this issue since there's nothing to fix here but feel free to continue the discussion about the release.

**Comment by xkr47** [comment]
My workaround until then; run this function once after loading jquery:
```js
function patchJqueryRecursionBug() {
  const orig = $.event.add
  $.event.add = function add(elem) {
    if (elem.nodeType === 3) return;
    return orig.apply($.event, arguments)
  }
}
```

**Comment by mgol** [comment]
That looks like a good workaround in the meantime, indeed!

**Comment by shenyu-li** [comment]
> Yes, this was fixed in #4558. We still have a few issues, also related to 3.4 focus changes that we'd like to fix in 3.5.0: https://github.com/jquery/jquery/milestone/21
> 
> I'll close this issue since there's nothing to fix here but feel free to continue the discussion about the release.

I have tried 3.5,1 and 3.5.0 and 3.4.1 and 3.4.0 and this issue 'Maximum call stack size exceeded' still exists for me, what is the fix?

**Comment by mgol** [comment]
> I have tried 3.5,1 and 3.5.0 and 3.4.1 and 3.4.0 and this issue 'Maximum call stack size exceeded' still exists for me, what is the fix?

then your issue is different, please open a new issue once you have a test case.
